tests/integration/nm/linux_bridge_test.py

```diff
@@ -53,6 +64,20 @@ def test_create_and_remove_bridge(eth1_up):
     assert not _get_bridge_current_state()
 
 
+@iproutelib.ip_monitor_assert_stable_link_up(BRIDGE0)
+def test_add_port_to_existing_bridge(bridge0_with_port0, eth2_up):
+    _add_port_to_bridge_config(bridge0_with_port0, (PORT1,))
+
+    _modify_bridge(bridge0_with_port0)
+
+    assert bridge0_with_port0 == _get_bridge_current_state()
+
+
+def _add_port_to_bridge_config(bridge0_with_port0, ports):
```

Comment:
IMHO reusing the name of the fixture, or is it the fixture here is confusing as well. If it is the fixture, the  function should be renamed to reflect this, if not, the argument should just be bridge. Also the functions  says `add_port` (singular) but the argument is `ports` (plural). Does it add only one port or also multiple ports?